woocommerce/wc-admin
# Add Home Screen Feedback admin note

Fixes #4493

This adds an admin note given the condition that it has been 12 days since the `woocommerce_homescreen_enabled` option has been set to `yes`.

The requirement was that the note be displayed 12 days after first accessing the new home page. Since there was no elegant way of determining that, I used the presence of the `woocommerce_homescreen_enabled` option instead.

### Screenshots

![image](https://user-images.githubusercontent.com/224531/84117158-68d5ed00-aa74-11ea-9513-f2c7ce84bee2.png)

### Detailed test instructions:

- Set the `woocommerce_homescreen_enabled` option to 'no' (or don't worry about it if the option doesn't exist yet)
- Refresh the page
- Verify that the `wc_admin_note_home_screen_feedback_homescreen_accessed` option has not been created
- Set the `woocommerce_homescreen_enabled` option to 'yes' (or create it if it doesn't exist)
- Refresh the page
- Verify that the `wc_admin_note_home_screen_feedback_homescreen_accessed` option has been created with the current timestamp
- Run the `wc_admin_daily` cron task
- Verify that the note has not been created
- Set the `wc_admin_note_home_screen_feedback_homescreen_accessed` option value back by at least 1036800 seconds (12 days)
- Run the `wc_admin_daily` cron task
- Verify that the note has been created
- Rejoice

### Changelog Note:

Enhancement: Add Home Screen Feedback admin note
